src/store/utils.js

```diff
@@ -51,6 +52,9 @@ export async function getPrices (baseCurrencies, toCurrency) {
   const { data } = await axios.get(`${COIN_GECKO_API}/simple/price?ids=${coindIds.join(',')}&vs_currencies=${toCurrency}`)
   let prices = mapKeys(data, (v, coinGeckoId) => findKey(cryptoassets, asset => asset.coinGeckoId === coinGeckoId))
   prices = mapValues(prices, rates => mapKeys(rates, (v, k) => k.toUpperCase()))
+  for (const code in nativeCodeSynonyms) {
```

Comment:
I think this would work the same if you use `baseCurrencies` to do the lookup on the prices returned from coingecko and then nativeCodeSynonyms would not be required

Reply:
I will try it, It should work I guess